storage-proofs/src/circuit/pedersen.rs

```diff
@@ -92,14 +93,18 @@ mod tests {
 
     #[test]
     fn test_pedersen_single_input_circut() {
+        let window_size = settings::SETTINGS
+            .lock()
+            .unwrap()
+            .pedersen_hash_exp_window_size;
         let mut rng = XorShiftRng::from_seed([0x5dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
 
         let cases = [(32, 689), (64, 1376)];
 
         for (bytes, constraints) in &cases {
             let mut cs = TestConstraintSystem::<Bls12>::new();
             let data: Vec<u8> = (0..*bytes).map(|_| rng.gen()).collect();
-            let params = &JubjubBls12::new();
+            let params = &JubjubBls12::new_with_window_size(window_size);
```

Comment:
instead of having the settings code + the `new_with_window_size` how about a single method in `storage-proofs` like `new_jubjub_bls12_current_window_size`, that does the work for us?